### DISCUSSION TITLE
Personal Blog made with Abell Starter

### DISCUSSION THREAD

**Discussion by haaarshsingh** [abelljs/abell#107] [Show your Abell Projects Here]
I've made my personal blog with Abelljs using the blog template! 

You can check out the website [here]
Here's the [source]

[here]: https://harshhhdev.github.io/blog
[source]: https://github.com/harshhhdev/blog

**Comment by iamsriranga** [comment]
@harshhhdev, that's nice, just a suggestion, do some work on responsiveness.

**Reply by haaarshsingh** [reply]
Yep, that's what I was going to work on next.

**Comment by saurabhdaware** [comment]
This looks amazing! Also I just checked the lighthouse score and it's 100! :D

**Reply by saurabhdaware** [reply]
ops it was 100 before the images I guess. If you resize images to match the size of the actual container, it will go back to 100 :D
